src/App/pages/Chart/Chart.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import { useRootStore } from '@context/StoreContext'
+import { IChartProduct } from '@store/models/chartProduct'
+import { observer } from 'mobx-react-lite'
+
+import styles from './Chart.module.scss'
+import { ChartList, ChartTotal } from './components'
+
+const Chart = () => {
+  const { chartStore } = useRootStore()
+
+  return (
+    <div className="container">
+      <div className={styles['chart-title']}>Chart</div>
+      <div className={styles['chart-content']}>
+        {chartStore.chartProducts.length ? (
+          <>
+            <ChartList
+              increaseCount={(id: number) => chartStore.increaseItemCount(id)}
+              decreaseCount={(id: number) => chartStore.decreaseItemCount(id)}
```

Comment:
не надо делать функции ради функций
```suggestion
              increaseCount={chartStore.increaseItemCount}
              decreaseCount={chartStore.decreaseItemCount}
```

Reply:
Я думал просто id не будет передаваться в таком случае, буду учитывать в будущем )